src/main.tsx

```diff
@@ -3,25 +3,43 @@ import React from "react";
 import ReactDOM from "react-dom/client";
 import { BrowserRouter, Route, Routes } from "react-router-dom";
 import "./global/default.css";
-import { GlucoseNotificationPrompt, Start, Welcome, RegistrationConfirmation } from "./routes";
+import {
+  GlucoseNotificationPrompt,
+  Start,
+  Welcome,
+  RegistrationConfirmation,
+  AddToHomescreen,
+} from "./routes";
 
 const client = new ApolloClient({
   uri: "http://localhost:8080/api/graphql",
   cache: new InMemoryCache(),
 });
 
+//test
+
 ReactDOM.createRoot(document.getElementById("root")!).render(
   <React.StrictMode>
     <ApolloProvider client={client}>
       <BrowserRouter>
         <Routes>
           <Route path="/" element={<div>Home Route</div>} />
-          <Route path="/registration-confirm" element={<RegistrationConfirmation />} />
-          <Route path="/onboarding/glucose-notifications" element={<GlucoseNotificationPrompt />} />
+          <Route
+            path="/registration-confirm"
+            element={<RegistrationConfirmation />}
+          />
+          <Route
+            path="/onboarding/glucose-notifications"
+            element={<GlucoseNotificationPrompt />}
+          />
+          <Route
+            path="/onboarding/add-to-homescreen"
```

Comment:
The ticket mentioned this to be a component. 
There shouldn't be a route to it since it is missing the drops icon and the set-up wizard.